dev/bots/test.dart

```diff
@@ -190,13 +190,18 @@ Future<Null> _analyzeRepo() async {
   await _checkForTrailingSpaces();
 
   // Try an analysis against a big version of the gallery.
+  // Generate into a directory that's not normally analyzed.
+  final String outDir = path.join(flutterRoot, 'bin', 'cache', 'dev', 'mega_gallery');
```

Comment:
Why are we even writing this into the flutter subdirectory tree?  If this is a test, we can use a temporary directory.

Reply:
Ah, this was historical from when the benchmark was being set up - it was easier to diagnose issues with the generated code. This is likely not needed now, so we could switch to something like a temp dir.